README.md

```diff
@@ -1,2 +1,47 @@
-# ui
-Reusable dApp UI components and setup functions
+# @gnosis.pm/dapp-ui
+<strong>Reusable Dapp UI components and setup functions</strong>
+
+## Setup
+As is probably already familiar, run:
+
+```sh
+npm i
```

Comment:
As on `dex-react`, should we stick to yarn over npm?

Reply:
Same, afterwards in a diff PR